testnet-node/docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       - "127.0.0.1:3334:3334"   # System API - binds to localhost:3334
       - "127.0.0.1:3335:3335"   # Prometheus API - binds to localhost:3335
       - "127.0.0.1:3336:3336"   # Engine State API - binds to localhost:3336
+      - "127.0.0.1:3337:3337"   # Mesh API - binds to localhost:3337
```

Comment:
Do we also want to turn it on in the `.env` file?

Reply:
I will turn it on, though not really sure if this really required to have it enabled by default